Mathlib/Data/Sym/Basic.lean

```diff
@@ -688,7 +688,8 @@ theorem encode_decode [DecidableEq α] (s : Sum (Sym (Option α) n) (Sym α n.su
     · refine' congr_arg Sum.inr _
       refine' map_injective (Option.some_injective _) _ _
       refine' Eq.trans _ (Eq.trans (SymOptionSuccEquiv.decode (Sum.inr s)).attach_map_coe _)
```

Comment:
Can you fit a `<;> simp` onto the previous line?

Reply:
I could, explointing that Lean accepts `.trans` for `Eq.trans`!